api/libproject/devstart.rb

```diff
@@ -253,11 +253,11 @@ def get_auth_login_account()
 
 def run_with_creds(args, command_name, proc)
```

Comment:
Is this being replaced by GcloudContext like below? (Are you planning on changing all other uses to do that?)

Reply:
Yes, I think I'd like to switch everything over so it's uniform, but wanted to just do a couple demo use cases to start.